examples/api/lib/cupertino/modal_bottom_sheet/modal_bottom_sheet.1.dart

```diff
@@ -0,0 +1,478 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+void main() {
+  runApp(const MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  const MyApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
```

Comment:
Same comment about maybe using CuperitnoApp and other Cupertino stuff as much as possible here. Unless maybe we want an example to show how to do this in Material.